administrator/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .user_profile import UserProfile
+from .user_profile import UserManager
```

Comment:
Maybe it's better to add a new line at the end of the file? This will prevent the line from being displayed as modified in future.